config.go

```diff
@@ -184,6 +184,7 @@ type Config struct {
 	RoleGroupDB ConfigRoleGroupDB
 	LDAP        ConfigLDAP
 	UserStore   ConfigUserStoreDB
+	AuditServiceUrl string
```

Comment:
Use an editor that runs gofmt when you save. One can see that this code has been edited with an editor that doesn't do that. Every reasonable Go editor I know of supports this.